ci/playstore/publish_to_internal_app_sharing.py

```diff
@@ -26,12 +28,32 @@
 
 # API calls
 
-apk_upload_request = androidPublisher.internalappsharingartifacts().uploadapk(packageName=package_name, media_body=apk_file, media_mime_type=None)
-apk_upload_response = apk_upload_request.execute()
-apk = {
+def upload_apk(package_name, apk_file):
+
+    apk_upload_request = androidPublisher.internalappsharingartifacts().uploadapk(packageName=package_name, media_body=apk_file, media_mime_type=None)
+    apk_upload_response = apk_upload_request.execute()
+    apk = {
         "version_name": version_name,
         "download_url": apk_upload_response['downloadUrl'],
         "certificate_fingerprint": apk_upload_response['certificateFingerprint']
     }
-
+    return apk
+
+def upload_apk_with_retry(package_name, apk_file, retry_count):
+
+    for i in range(0, retry_count - 2):
+        while True:
+            try:
+                apk = upload_apk(package_name, apk_file)
+                return apk
+            except HttpError as err:
+                if err.resp.status in [403, 500, 503]:
+                    time.sleep(5)
+                else: raise
+                continue
+            break
+        apk = upload_apk(package_name, apk_file)
+        return apk
+
+apk = upload_apk_with_retry(package_name, apk_file, 3)
```

Comment:
Maybe I will move the 3 to a constant value to know that are the retry times at a glance